### ISSUE TITLE
"Pick up and deliver mode"

### ISSUE THREAD

**Issue by Murtlock** [Stephan-S/FS19_AutoDrive#471]
Hi Stephan,
In this mode the unit does not stop fully to load and then drive of.
Instead its just passing trough

**Comment by Stephan-S** [comment]
Hm, it's working perfectly fine on my tests.
What is your setup? What vehicle/trailers are you trying to fill?
What are you trying to fill and from what type of silo?

**Comment by Murtlock** [comment]
hi.
its normal trailer agroliner tdk 302 and im trying to load sugar and flour from fed mods sugar and flour fabricks in northfrische march 2.3

**Comment by Stephan-S** [comment]
And did this work in earlier versions of Autodrive with these factories?
Could be an issue with the factory triggers

**Comment by Murtlock** [comment]
yes it did ill try something then will see

**Comment by Murtlock** [comment]
now it freezes my whole game :(

**Comment by ridernl** [comment]
Same problem here, there is no error log tho, it just keeps on driving.
7210pro tractor with a sek 802 (extended to 12000l capacity) trailer.

Did work fine in previous versions, even went back a version to make sure if it would behave.
Only happend with the latest

**Comment by Bobster82** [comment]
> Since i also have a quite similar problem i wont open another issue.
> 
> I have a waypoint where i want to stop a tipper and wait to be loaded by a wheelloader but the driver always slows down, stops and continue driving to the destination without waiting to be filled.
> Options are set to wait on empty silo.

Got the same problem here, wont stop with the pick up and deliver option

**Comment by Stephan-S** [comment]
Should all be working fine in the current version now. Can you confirm?

**Comment by ridernl** [comment]
Confirmed it is working fine with the new version :)
Thank you!

**Comment by Stephan-S** [comment]
Thank you for the confirmation.
